lib/shared/config/loadfiles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var get = require('lodash.get');
```

Comment:
I'm going to be removing lodash. It's okay for us to leave it in for this implementation unless you want to find replacements